Formula/mvnvm.rb

```diff
@@ -9,14 +9,14 @@ class Mvnvm < Formula
 
   depends_on :java => "1.7+"
 
+  conflicts_with "maven", :because => "also installs a 'mvn' executable"
```

Comment:
right.
```
"also installs a `mvn` executable"
```

You'll notice there's a mixture of backticks and single quotes throughout these but let's try to standardize on backticks within the `:because` lines. I'd be happy with single quotes everywhere instead but I think most of the existing ones use backticks:
```
Josephs-MacBook-Pro:Formula joe$ ag ':because'|grep -v "\`"|grep "'"|wc -l
      16
Josephs-MacBook-Pro:Formula joe$ ag ':because'|grep "\`"|grep -v "'"|wc -l
     126
Josephs-MacBook-Pro:Formula joe$ ag ':because'|grep -v "\`"|grep -v "'"|wc -l
      69
```